LandSandBoat/server
# Update Sange & Yonin to better match retail

**_I affirm:_**

## What does this pull request do?

Fixes Yonin enmity value & decay to match retail
Fix Sange animation
Convert Sange to retail's Sange, including augments
Adds some TODOs for barrage enhancement
fixes a typo or two

Sange not giving +racc at 5/5 merits directly (therefore the +racc must be applied only to daken invisibly):
![sange racc doesnt boost](https://github.com/user-attachments/assets/b4f4a646-9a7a-4040-ab0b-cf3c4550907c)

Sange animation capture:
![sange](https://github.com/user-attachments/assets/05b8cc81-706b-4f6f-a5b6-9fa41d28c9f9)


## Steps to test these changes

Use sange, use yonin, see them work correctly

With Sange, you'll need to breakpoint in the attacks to see the additional merits/augs work
To get a correct Koga Chainmail+2 do the following:
![image](https://github.com/user-attachments/assets/9cd9981f-6536-4fac-b157-c8f247f54a2c)
